core/src/test/java/org/kohsuke/stapler/DispatcherTest.java

```diff
@@ -359,6 +989,4 @@ public void testPublicFieldDispatch() throws Exception {
     }
 
 
-
-
-}
\ No newline at end of file
+}
```

Comment:
Very nice set of tests here. Perhaps just split into multiple class to ease the readability ?